### PR TITLE
K2v bridge

### PR DESCRIPTION
**Pull request by jonaphael**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: ansible-dojot/roles/k2v-bridge/templates/k2v_bridge_cluster.yaml.j2 [hunk 1]
```diff
@@ -31,10 +31,14 @@ spec:
...
         - name: CERT_DNS
-          value: "mqtt-client"
+          value: "k2v-bridge"
         - name: CERT_EJBCA_API_BROKER
           value: "ejbca-simple"
         - name: CERT_EJBCA_API_PORT
           value: "5583"
+        - name: ENABLE_DOJOT
+          value: "false"
+        - name: MQTT_CLIENT_LOG_LEVEL
```
Comment (mprevide) [inline ansible-dojot/roles/k2v-bridge/templates/k2v_bridge_cluster.yaml.j2]: Acho que nao precisa do prefixo, poderia ser só log_level

**Comment by mprevide** [comment]
No arquivo app.test.ts.dev, a extensão é pra ser .dev mesmo?

**Comment by jonaphael** [comment]
> No arquivo app.test.ts.dev, a extensão é pra ser .dev mesmo?

arquivo removido

File: k2v-bridge/src/app.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+const { logger } = require('@dojot/dojot-module-logger');
+const { IoTAgent } = require('@jonaphael/iotagent-nodejs');
+const fs = require('fs');
+const mqtt = require('mqtt');
+const util = require('util');
+const ProjectUtils = require('../utils/utils');
+const defaultConfig = require('./config');
+
+const TAG = { filename: 'MqttClientApp' };
+const hostname = process.env.HOSTNAME;
```
Comment (mprevide) [inline k2v-bridge/src/app.js]: Acho que é melhor estar no arquivo de configuração :)

File: k2v-bridge/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+# **Configurations**
+
+## **Environment Variables**
+
+Before running the kubernetes environment with the k2v-bridge service, make sure you configure the environment variables to match your needs. The variables can be configured in the .yaml file of the service.
+
+Key                      | Purpose                                                             | Default Value   | Valid Values   |
+------------------------ | ------------------------------------------------------------------- | --------------- | -------------- |
+MQTT_CLIENT_LOG_LEVEL    | Log level for debug                                                 | info            | string         |
```
Comment (mprevide) [inline k2v-bridge/README.md]: Renomear pra LOG_LEVEL

**Pull request merged**